cli/smartcontract/smart_contract.go

```diff
@@ -135,7 +136,17 @@ func initSmartContract(ctx *cli.Context) error {
 	// TODO: Fix the missing neo-go.yml file with the `init` command when the package manager is in place.
 	if !ctx.Bool("skip-details") {
 		details := parseContractDetails()
-		if err := ioutil.WriteFile(filepath.Join(basePath, "neo-go.yml"), details.toStormFile(), 0644); err != nil {
+		details.ReturnType = rpc.ByteArray
+		details.Parameters = make([]rpc.StackParamType, 2)
+		details.Parameters[0] = rpc.String
+		details.Parameters[1] = rpc.Array
+
+		project := &ProjectConfig{Contract: details}
+		b, err := yaml.Marshal(project)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+		if err := ioutil.WriteFile(filepath.Join(basePath, "neo-go.yml"), b, 0644); err != nil {
```

Comment:
nit: probably next time move "neo-go.yml" to variable or config

Reply:
It's an init command that creates a new directory with some default setup, so it's OK for it to have this file name hardcoded, making it configurable gives nothing to the user.

Reply:
for user sure, but for developer it can give readability and idea why there is such name.